lib/pkcs7.js

```diff
@@ -328,8 +328,26 @@ p7.createSignedData = function() {
 
     /**
      * Signs the content.
+     *
+     * Includes the signed data with digital signature.
      */
     sign: function() {
+      msg.baseSign(false);
+    },
+
+    /**
+     * Signs the content in detached mode
+     *
+     * Includes the signature and certificate without the signed data.
+     */
+    signDetached: function() {
+      msg.baseSign(true);
+    },
+
+    /**
+     * Signs the content.
+     */
+    baseSign: function(detached) {
```

Comment:
Should this be `_baseSign`?  Do we want this API to be part of the public API?

Reply:
Fixed in https://github.com/digitalbazaar/forge/pull/605/commits/09afde45c47fa21f58038bf1685ff06b9410ceb2